src/redux/studio-permissions.js

```diff
@@ -35,7 +35,7 @@ const selectCanDeleteCommentWithoutConfirm = state => selectIsAdmin(state);
 const selectCanFollowStudio = state => selectIsLoggedIn(state);
 
 // Matching existing behavior, only admin/creator is allowed to toggle comments.
-const selectCanEditCommentsAllowed = state => !selectIsMuted(state) && (selectIsAdmin(state) || isCreator(state));
+const selectCanEditCommentsAllowed = state => !selectIsMuted(state) && (selectIsAdmin(state) || isHost(state));
```

Comment:
I'm confused why a muted studio host can't close comments. If comments need to be closed due to spam or anything else of the sort, actions should be able to be taken immediately, and a mute doesn't seem to be needed in this case.